app/templates/reports/manage_report.html

```diff
@@ -30,6 +30,13 @@
         <tr><td>Duration</td><td>{{ report.duration}}</td></tr>
         <tr><td>Agency</td><td>{{ report.agency.name }}</td></tr>
         <tr><td>Picture</td><td><img src="{{ report.picture_url }}" alt="report picture" width="200"</td></tr>
+        <tr><td>Weather conditions during incident</td>
```

Comment:
I think we can just call it "Weather Condition" since we know it's already for an incident report